helix-term/src/ui/editor.rs

```diff
@@ -66,6 +65,52 @@ impl EditorView {
         &mut self.spinners
     }
 
+    #[allow(clippy::too_many_arguments)]
```

Comment:
I had to add this one allow attribute here to make clippy shutup,
This function should be a closure that captures all arguments except `base`.
However the function must be generic over `base` but that is currently not possible with closures.
So the reason for the large argument count is just that were working around limitations of the rust type system.

Reply:
not needed anymore after 2c6f681